src/controllers/v1.controller.ts

```diff
@@ -173,7 +175,16 @@ export class V1Controller {
 
       // Fetch applications contained in this Load Balancer. Verify they exist and choose
       // one randomly for the relay. First check RPC ID to see if this client should be stuck to an app and node.
-      const { preferredApplicationID, preferredNodeAddress, rpcID } = await this.checkClientStickiness(rawData)
+
+      const { stickiness = false, stickinessDuration = DEFAULT_STICKINESS_DURATION } = loadBalancer
+
+      const { preferredApplicationID, preferredNodeAddress, rpcID } = stickiness
+        ? await this.checkClientStickiness(rawData)
+        : {
+            preferredApplicationID: '',
+            preferredNodeAddress: '',
+            rpcID: 0,
+          }
```

Comment:
I'd prefer it this way; but feel free to keep it like it is now as the logic is the same.
```suggestion
      const stickinessData = stickiness
        ? await this.checkClientStickiness(rawData)
        : {
            preferredApplicationID: '',
            preferredNodeAddress: '',
            rpcID: 0,
          }

      const { preferredApplicationID, preferredNodeAddress, rpcID } = stickinessData
```

Reply:
For me the original way reads fine and is possibly clearer.